src/notify.js

```diff
@@ -59,6 +63,14 @@ function Notify(title, options = {}) {
     }
 }
 
+Notify.serviceWorkerSupported = function() {
```

Comment:
How about we combine this with a feature detect for push manager, then we could call it `isPushNotificationSupported()` and use it as suggested in the comment below.